dvc/repo/remove.py

```diff
@@ -11,9 +11,14 @@
 
 @locked
 def remove(self: "Repo", target: str, outs: bool = False):
-    stages = self.stage.from_target(target)
+    stages_info = self.stage.collect_granular(target)
```

Comment:
Does the `target` argument description string (`argparse`) need an update? 🙂

Reply:
Indeed, although I think I should wait for some clarifications regarding the doc update (https://github.com/iterative/dvc.org/pull/2357) before updating the argument description